client/core/core_test.go

```diff
@@ -76,6 +76,7 @@ var (
 	tSwapFeesPaid       uint64 = 500
 	tRedemptionFeesPaid uint64 = 350
 	tLogger                    = dex.StdOutLogger("TCORE", dex.LevelTrace)
+	tNotified                  = false
```

Comment:
This shouldn't be a package-level var. Use a `NotificationFeed` in the test function, and look for only the notification you want.

Reply:
I'm using NotificationFeed now.